Documentation/ApiOverview/DataFormats/T3datastructure/Elements/Index.rst

```diff
@@ -21,6 +20,12 @@ contain another set of elements.
 (In a PHP array this corresponds to saying that all these elements
 must be arrays.)
 
+..  versionchanged:: 12.0
+    The superfluous array key `TCEforms` was removed and is not evaluated
+    anymore. Its sole purpose was to wrap real TCA definitions.
+
+    The tags `TCEforms` **should** be removed upon dropping TYPO3 v11 support.
```

Comment:
Maybe move this to the sentence above as it looks like not belong to the "versionchanged" part above in rendered doc.
We should also mention that in v12 a migration is in place, which will be removed in future TYPO3 versions.